### DISCUSSION TITLE
Wifi 6 command-> how to use

### DISCUSSION THREAD

**Discussion by henkiejan1** [arendst/Tasmota#21639] [Q&A] [answered]
Hi,

With the introduction of Tasmota Rodney i read the following text in the changelog:
`Command Wifi 6 to enable 11ax on ESP32`

But when in try to use it on a ESP32 C6 on the console i got only the following response and nothing happen...

`14:08:09.586 CMD: wifi 6
14:08:09.588 RSL: RESULT = {"Wifi":"ON","Mode":"HE20"}`

Do i something wrong or is it just not working??

**Comment by Jason2866** [comment, marked as answer]
You have to configure your AP to force wifi 6. You can only enable the mode on Tasmota side. The closed source wifi stack (not under Tasmota control) "decides" in which mode it is connecting.
AND there are APs and Routers which supports mode 11ax only with 5GHz.
